pom.xml

```diff
@@ -283,6 +261,15 @@
         </repository>
     </repositories>
     <profiles>
+        <profile>
+            <id>JDK17</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
You can add 

```
<jdk>17</jdk>
```

And the profile will activate based on JDK, which is what we want.